BSP/Simulator_V2/Src/BSP_CAN.c

```diff
@@ -85,6 +104,9 @@ ErrorStatus BSP_CAN_Write(uint32_t id, uint8_t data[8], uint8_t length) {
  * @return  ERROR if nothing was received so ignore id and data that was received. SUCCESS indicates data was received and stored.
  */
 ErrorStatus BSP_CAN_Read(uint32_t *id, uint8_t *data) {
+    char buffer[50];
+    sprintf(buffer, "Read CAN message of ID 0x%X\n", *id);
```

Comment:
Why do you print the ID but not the data?

Reply:
See CAN_READ comments - would need to get actual CAN messages to be "transmitted" and then read into those pointers. My current code doesn't really do that because there's nowhere to grab a message from unless I use the custom FIFO, I think